includes/gateways/manual.php

```diff
@@ -32,7 +31,7 @@
  * @return bool
  **/
 function give_manual_form_ouput() {
```

Comment:
@henryholtgeerts Typo in function name

Reply:
fixed: https://github.com/impress-org/givewp/compare/66114f9b23fd...c5481037f8ec